litmus/director/TCID-DIR-OP-DELETE-EXTERNAL-CSPC-WITH-NO-VOLUME/test.yml

```diff
@@ -33,7 +33,7 @@
 
         ## Get node name
         - name: Get node name
```

Comment:
@amitbhatt818 you are getting Bd name or node name

Reply:
I will change the comments @nsathyaseelan